### PR TITLE
Minor Improvements

### PR DESCRIPTION
**Pull request by sat3ll**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,9 @@ python video2x.py -i sample_input.mp4 -o sample_output.mp4 -m cpu --width=1920 -
 ### -h, --help
     show this help message and exit
 
+### -y
+    Answer automatically all questions
```
Comment (k4yt3x) [inline README.md]: Grammar sounds a bit strange.

> Automatically answer all questions.

File: bin/video2x.py [hunk 2]
```diff
@@ -83,10 +83,11 @@ def process_arguments():
...
     upscaler_options.add_argument('-m', '--method', help='upscaling method', action='store', default='gpu', choices=['cpu', 'gpu', 'cudnn'])
     upscaler_options.add_argument('-d', '--driver', help='waifu2x driver', action='store', default='waifu2x_caffe', choices=['waifu2x_caffe', 'waifu2x_converter', 'waifu2x_ncnn_vulkan'])
-    upscaler_options.add_argument('-y', '--model_dir', help='directory containing model JSON files', action='store')
+    upscaler_options.add_argument('--model_dir', help='directory containing model JSON files', action='store')
     upscaler_options.add_argument('-t', '--threads', help='number of threads to use for upscaling', action='store', type=int, default=1)
     upscaler_options.add_argument('-c', '--config', help='video2x config file location', action='store', default=os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])), 'video2x.json'))
-    upscaler_options.add_argument('-b', '--batch', help='enable batch mode (select all default values to questions)', action='store_true')
+    upscaler_options.add_argument('-y', help='automatically answer to all questions', action='store_true', dest='auto_answer')
```
Comment (k4yt3x) [inline bin/video2x.py]: grammar change

File: bin/video2x.py [hunk 3]
```diff
@@ -292,28 +296,32 @@ def absolutify_paths(config):
...
-            else:
-                Avalon.error(f'Unable to create {video2x_cache_directory}')
-                Avalon.error('Aborting...')
-                exit(1)
-    else:
-        Avalon.error('Specified cache directory is a file/link')
-        Avalon.error('Unable to continue, exiting...')
-        exit(1)
+    video2x_cache_directory = tempfile.mkdtemp(prefix='video2x-')
+else:
```
Comment (k4yt3x) [inline bin/video2x.py]: these two lines can be combined into one

File: bin/video2x.py [hunk 4]
```diff
@@ -359,7 +368,15 @@ def absolutify_paths(config):
     elif os.path.isdir(args.input):
         # upscale videos in a directory
         Avalon.info(f'Upscaling videos in directory: {args.input}')
+
+        # determine which files are videos
+        accepted_formats = ['.mkv', '.mp4', '.webm']
```
Comment (k4yt3x) [inline bin/video2x.py]: more formats are accepted

Comment (sat3ll) [reply bin/video2x.py]: I can't find a comprehensive list of formats available, I only listed the most common. `ffmpeg -formats` does not do the trick.

File: bin/video2x.py (hunk 2 above)
Comment (sat3ll) [reply bin/video2x.py]: should I reuse the --batch mode text to the -y flag?

**Comment by k4yt3x** [comment]
Some of the changes are already adopted in the newer versions of Video2X. Others will be added after the Linux compatibility release.

**Pull request closed without merging**